readme.md

```diff
@@ -56,6 +41,63 @@ Import certificate into truststore:
         -file doej.cer \
         -keystore doej.truststore
 
+## Gradle
+
+If you are already familiar with gradle, or have it installed on your system feel free to skip ahead to the next section in the documentation.
+
+### The Gradle Wrapper
+
+Once you have cloned the skeleton, generated keys, and changed into its directory, you can build and run it by typing the following into the command line
+
+    $ ./gradlew build
+    $ ./gradlew run
+
+While this works fine, you can save yourself the './' and the 'w' by installing gradle and adding it to your path.
+
+### Installing Gradle
+
+Using your favorite web browser travel to 'gradle.org/gradle-download' and download version 2.4 from the previous releases section in the lower right. You only need to download the binary only, but if you want the documentation you should download the complete version. Now:
+
+    $ cd /path/to/gradle-2.4-bin.zip
+    $ unzip gradle-2.4-all.zip -d /path/to/destination_folder
+
+Now to install gradle
+
+    $ cd /path/to/gradle-2.4/bin
+    $ gradle
+
+If gradle successfully installed you will see a message with the gradle version and some other information. Now you can add it to your path.
+
+### Adding Gradle to your path
+
+These next steps assume you are using bash, and if you are will add gradle to your path. You can use your favorite text editor, but this documentation uses vim.
+
+    $ vim /home/your_username/.bashrc
```

Comment:
it would be sufficient to say something like "edit the file `foo` and add the line `bar`" since which editor is not important.